test/CadlRanchProjects/encode/datetime/src/Generated/Docs/Header.xml

```diff
@@ -7,15 +7,15 @@ This sample shows how to call DefaultAsync.
 <code><![CDATA[
 Header client = new DatetimeClient().GetHeaderClient();
 
-Response response = await client.DefaultAsync(DateTimeOffset.Parse("2022-05-10T14:57:31.2311892-04:00"));
```

Comment:
isn't this a breaking change if it happens in serialization scenarios?

Reply:
Because of the `InputParameterGetSerializationFormat`, there is no change in serialization code - only in samples. However, if we switch HLC to input model, there will be breaking changes in serialization, hence the PR to avoid them.

Reply:
Sure: https://github.com/Azure/azure-sdk-for-net/pull/42602/files